### ISSUE TITLE
Brew issue with 1.0.1

### ISSUE THREAD

**Issue by JasonIamAUnixAdmin** [disneystreaming/ssm-helpers#51]
There is an issue upgrading to 1.0.1 via brew. I am consistently getting the following error:

```
==> Upgrading disneystreaming/tap/ssm-helpers
  1.0.0 -> 1.0.1

==> Downloading https://github.com/disneystreaming/ssm-helpers/releases/download/v1.0.1/ssm-helpers_1.0.1_Darwin_x86_64.tar.gz
==> Downloading from https://github-releases.githubusercontent.com/222777077/8ad95680-d05c-11eb-8f7c-1302257fd616?X-Amz-Algorithm=AWS4-HMAC-SHA256&X
######################################################################## 100.0%
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - ssm-helpers
  ```
  checking the zip by hand the files inside seem fine. 
  
  running in debug you see:
  ```
  tar --extract --no-same-owner --file /Users/jasonschuster/Library/Caches/Homebrew/downloads/e5ed3b37a2e552746bf1769a2d2b1b8cd21346c658cf62a4198e6e3d1319bc9e--ssm-helpers_1.0.1_Darwin_x86_64.tar.gz --directory /private/tmp/d20210624-35277-17dlyqm
cp -pR /private/tmp/d20210624-35277-17dlyqm/ssm /private/tmp/ssm-helpers-20210624-35277-1b4n2pl/ssm
cp -pR /private/tmp/d20210624-35277-17dlyqm/LICENSE /private/tmp/ssm-helpers-20210624-35277-1b4n2pl/LICENSE
cp -pR /private/tmp/d20210624-35277-17dlyqm/README.md /private/tmp/ssm-helpers-20210624-35277-1b4n2pl/README.md
/usr/local/Homebrew/Library/Homebrew/ignorable.rb:29:in `block in raise'
Errno::ENOENT: No such file or directory - ssm-helpers
  ```

Thanks for a great tool that makes life better most every day.

**Comment by rothgar** [comment]
I think it may be coming from this in the formula that looks like it was reverted to installing the `ssm-helpers` binary instead of `ssm` https://github.com/disneystreaming/homebrew-tap/blob/master/Formula/ssm-helpers.rb#L29

I think that's defined in .goreleaser.yaml but I'm not 100% sure.

**Comment by sendqueery** [comment]
Fixed in #52! 🎉

**Comment by JasonIamAUnixAdmin** [comment]
Thank you!
